ui/package.json

```diff
@@ -5,7 +5,10 @@
   "dependencies": {
     "react": "^16.8.0",
     "react-dom": "^16.8.0",
-    "react-scripts": "2.1.3"
+    "react-scripts": "2.1.3",
+    "redux": "^4.0.1",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.3.0"
   },
```

Comment:
Is a good idea to use `^` here? Maybe, is not better it be `~`? Wdyt?

Reply:
Sorry, I saw now that it is in WIP. The rest of the code shows great anyway.

Reply:
@camilamacedo86 np :) re versions these will be locked down before PR is merged